src/output/icons.rs

```diff
@@ -7,103 +7,103 @@ use crate::fs::File;
 struct Icons;
 
 impl Icons {
```

Comment:
Can we turn if off just for this list of constants?

Reply:
yep, probably ... gonna skim the code for those places and try to use rustfmt's skips there.

Reply:
fixed